blob/s3blob/s3blob.go

```diff
@@ -144,24 +143,58 @@ type URLOpener struct {
 	Options Options
 }
 
+const (
+	sseTypeParamKey  = "ssetype"
+	kmsKeyIdParamKey = "kmskeyid"
+)
+
+func IsValidServerSideEncryptionType(value string) bool {
```

Comment:
This doesn't need to be exported -> isValidServerSideEncryptionType

Reply:
Actually let's change it to "toServerSideEncryptionType" and have it return (typesv2.ServerSideEncryption, error).